antora-ui-camel/src/css/frontpage.css

```diff
@@ -74,29 +74,30 @@ header.frontpage p {
   line-height: 1.7rem;
 }
 
-.frontpage a.button-dark,
-.frontpage a.button-dark:hover {
+a.button-dark,
+section.frontpage a.button-dark,
+section.frontpage a.button-dark:hover,
+a.button-dark:hover {
   white-space: nowrap;
   padding: 0.7rem 1rem;
```

Comment:
```suggestion
  padding: 0.3rem 1rem;
```

I think smaller buttons look a bit less imposing:

![Screenshot_2020-08-24 Documentation - Apache Camel](https://user-images.githubusercontent.com/1306050/91084066-b0b7e980-e64b-11ea-9a66-3c5ce3ee1db9.png)